### ISSUE TITLE
The template project doesn't work

### ISSUE THREAD

**Issue by mrj0** [enhance-dev/enhance-starter-project#42]
From the readme:

```
$ npx "@enhance/create@latest" ./myproject -y

Failed to create! ./myproject

Invalid template URL
```

**Comment by brianleroux** [comment]
can confirm! having a look now 

cc @kristoferjoseph @macdonst 

(aside eh guys we should deprecate to remove enhance-dev/create ?)

**Comment by brianleroux** [comment]
@mrj0 we have a pathing issue… not quite sure whats up but if you run `npm i -g @enhance/cli` you can use the enhance cli to generate project resources in the meantime

**Comment by macdonst** [comment]
Ug, the package is treating `-y` as the template URL. Running:

```
npx "@enhance/create@latest" ./myproject
```

Should just work. 

I will look into this bug later today.

**Comment by macdonst** [comment]
I just published a fix in create 4.1.4.
